### PR TITLE
Kazoo script

### PR DESCRIPTION
**Pull request by lazedo**
* allow per node vm.args/sys.config
* remove code path as its not needed

### CODE REVIEW

File: rel/ci.vm.args [hunk 1]
```diff
@@ -21,7 +21,7 @@
 ##-heart
 
 # GC more often
--env ERL_FULLSWEEP_AFTER 10
+# -env ERL_FULLSWEEP_AFTER 10
```
Comment (k-anderson) [inline rel/ci.vm.args]: @lazedo lets remove the line rather than comment out.  Git has a history feature ;)

File: rel/kazoo [hunk 2]
```diff
@@ -40,6 +40,7 @@ ERL_OPTS="{{ erl_opts }}"
 RUNNER_LOG_DIR="${RUNNER_LOG_DIR:-/tmp/log}"
 VM_NODE_FILES_DIR="${VM_NODE_FILES_DIR:-/tmp}"
 NAME_TYPE="${NODE_NAME_TYPE:--name}"
+NODE_STARTUP_FILE_PATH=${NODE_STARTUP_FILE_PATH:-/etc/kazoo/core}
```
Comment (k-anderson) [inline rel/kazoo]: @lazedo as per the standup, this looks fine but makes me nerves and we agreed I owe you a shot as well as a beer (at least) if someone needs this functionality.  ;)

Comment (lazedo) [reply rel/kazoo]: i pushed GC change in different PRs. the 4.3 of this PR should be merged after next release.

File: rel/ci.vm.args (hunk 1 above)
Comment (lazedo) [reply rel/ci.vm.args]: pushed in #5628 & #5629

**Pull request merged**